tasks/section5.yml

```diff
@@ -571,15 +743,19 @@
       - rule_5.4.1.1
 
 - name: "SCORED | 5.4.1.1 | PATCH | Ensure password expiration is 90 days or less"
-  command: /bin/true
+  lineinfile:
+      state: present
+      dest: /etc/login.defs
+      regexp: '^PASS_MAX_DAYS'
+      line: 'PASS_MAX_DAYS 90'
   tags:
       - level1
       - level2
       - patch
       - rule_5.4.1.1
 
 - name: "SCORED | 5.4.1.2 | AUDIT | Ensure minimum days between password changes is 7 or more"
-  command: /bin/true
+  command: grep PASS_MIN_DAYS /etc/login.defs
   register: result
```

Comment:
Audit tasks should register verbose variable name that ends with `_audit`. As you are adding AUDIT checks please change `register: result` to `register: `*`descriptive_check_name_audit`*